blob/fileblob/fileblob.go

```diff
@@ -16,10 +16,12 @@
 // filesystem. This should not be used for production: it is intended for local
 // development.
 //
-// Blob names must only contain alphanumeric characters, slashes, periods,
-// spaces, underscores, and dashes. Repeated slashes, a leading "./" or "../",
-// or the sequence "/./" is not permitted. This is to ensure that blob names map
-// cleanly onto files underneath a directory.
+// Blob names are escaped using url.PathEscape before writing them
```

Comment:
Let's do this as a separate change: there's enough other business going on in this PR.

Reply:
Done, sent PR #588.